### ISSUE TITLE
Dependency lib fails to install on node version v10.15.3

### ISSUE THREAD

**Issue by vitorhsb** [vercel/next.js#20168]
# Bug report

## Describe the bug

Sharp lib fails to install on node version v10.15.3. This is due to v10.16 requirement by the sharp repository: https://github.com/lovell/sharp/blob/c10888e6fe38885e2a21ee9598b941001920fd22/docs/changelog.md#v0260---25th-august-2020 starting on v0.26.0 - 25th August 2020

## To Reproduce

Install any nextjs example with node version below v10.16 .

## Expected behavior

All project dependencies should install as expected on version >= 10.0.3 since this is the supported version as stated in https://github.com/vercel/next.js README badge.

## Screenshots

![image](https://user-images.githubusercontent.com/86801/102084129-98432900-3e0c-11eb-9403-9f7581ab6ec7.png)

## System information

- OS: macOS
- Version of Next.js: 10.0.3
- Version of Node.js: 10.15.3
- Deployment:

```
npx create-next-app --example basic-css basic-css-app
cd basic-css-app
npm i
```

**Comment by vitorhsb** [comment]
I've now seen that Next package.json indicates that sharp is an optionalDependency and it will not break repository install.

But does it mean that built-in Image Optimization is disabled?

If so, this error page should be updated to explicitly declare required node version >= 10.16.0
https://github.com/vercel/next.js/blob/canary/errors/install-sharp.md

And documentation must be update to explicitly declare required node version >= 10.16.0
https://nextjs.org/docs/basic-features/image-optimization

**Comment by diegofss11** [comment]
@vitorhsb , either install `sharp` as a dependency or update your node version to `10.16.0`. If my understanding is correct, `sharp` is now necessary for the newer Next.js version due to the image optimisations.

**Comment by vitorhsb** [comment]
I have just notice that the `build` command fails when the `sharp` dependency is not available starting on version v10.0.4 (Issue linked: https://github.com/vercel/next.js/issues/20456).

This enforces that documentation must be updated to reflect this node version dependency.

> If so, this error page should be updated to explicitly declare required node version >= 10.16.0
> https://github.com/vercel/next.js/blob/canary/errors/install-sharp.md
>
> And documentation must be update to explicitly declare required node version >= 10.16.0
> https://nextjs.org/docs/basic-features/image-optimization

Also, the Next.js project (README.md at least) must reflect this required node version:

![image](https://user-images.githubusercontent.com/86801/103807536-6c534600-504e-11eb-9e7b-9bc803b7c616.png)

**Comment by vitorhsb** [comment]
Starting from Next.js version 10.0.8 `sharp` dependency is no longer necessary.

Closing since this is not an issue anymore.
